### PR TITLE
fix(label): use dark color for checkbox and radio

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [ ] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [ ] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by mohamedMok** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [a75f83fd\-83ae\-464b\-b96c\-aac911488367](https://eu.ast.checkmarx.net/projects/d2ea2837-0ed1-4404-8c6a-f1a1376ffd05/scans?id=a75f83fd-83ae-464b-b96c-aac911488367&branch=fix-color-label)
### No New Or Fixed Issues Found

**Review by matthieu-castier — APPROVED** [review]
I tested this modifications on my project and it's good for me.

File: packages/styles/components/_c.checkbox.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     cursor: pointer;
     margin-left: magic-unit-rem(0.5, true);
+    color: $color-font-darker;
```
Comment (tiloyi) [inline packages/styles/components/_c.checkbox.scss:15]: According to the Figma mockups, I think that the right color is: `$color-font-darkest` that is the equivalent of `#000`

File: packages/styles/components/_c.radio.scss [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
     cursor: pointer;
     margin-left: magic-unit-rem(0.5, true);
+    color: $color-font-darker;
```
Comment (tiloyi) [inline packages/styles/components/_c.radio.scss:15]: According to the Figma mockups, I think that the right color is: `$color-font-darkest` that is the equivalent of `#000`

**Review by matthieu-castier — APPROVED** [review]

**Review by tiloyi — APPROVED** [review]

**Pull request merged**